### ISSUE TITLE
Current geanypy no longer seems to pick up plugins in folders

### ISSUE THREAD

**Issue by olymk2** [codebrainz/geanypy#42]
On ubuntu 16.04 at least the newer geany and geanypy do not seems to detect plugin that are placed inside a folder, previously this used to work.

Has this been fixed at all or is anyone aware of the issue ?

**Comment by kugel-** [comment]
Did you install geanypy with "apt-get install geany-plugins"? If so, then this is expected. plugins must be moved to the normal plugin location where geany can find them since geanypy doesn't locate them anymore itself.

Note, this is only applies to the geanypy version at [geany-plugins](https://github.com/geany/geany-plugins). This (upstream) geanypy doesn't have that change yet, see #37

**Comment by codebrainz** [comment]
@kugel- I thought you fixed that so it would also find old plugins, no?

**Comment by kugel-** [comment]
No. How could I possibly workaround it if the lookup happens outside geanypy? Also, the release notes mentioned this IIRC.

**Comment by codebrainz** [comment]
@kugel- I was probably thinking of [this](https://github.com/codebrainz/geanypy/pull/37#issuecomment-154690610) where you said:

> all existing plugins continue to work as-is, without any changes

The only way I can think of if GeanyPy isn't loading any more is for GeanyPy to detect that user has plugins in the old directory and offer to move them (or at least give instructions on where to move them to).

**Comment by olymk2** [comment]
whats considered a normal plugin location ? 
do you mean ~/.config/geany/plugins/mypythonplugin/pluginfiles.py 
I will give it a try later cheers

**Comment by kugel-** [comment]
~/.config/geany/plugins/mainfile.py, other files can be in a subdirectory using normal python mechanisms

**Comment by olymk2** [comment]
okay, thats why all my plugins broke because they where each self contained in there own folder, i will restructure them with a main file outside cheers for clarifying
